src/app/home/form/form.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import firebase from 'firebase/app';
+import 'firebase/auth'
+import {environment} from '../../../environments/environment'
+
+@Injectable()
+export class FormService {
+
+  constructor(private firebaseConfig: object) {
+    firebaseConfig = {
+      apiKey: environment.firebase.apiKey,
+      authDomain: environment.firebase.authDomain,
+      projectId: environment.firebase.projectId,
+      storageBucket: environment.firebase.storageBucket,
+      messagingSenderId: environment.firebase.messagingSenderId,
+      appId: environment.firebase.appId,
+      measurementId: environment.firebase.measurementId,
+    };
+    firebase.initializeApp(this.firebaseConfig);
```

Comment:
Does `form.service.ts` seems an appropriate name to you? Should we replace it with something like `firebase-intializer.service.ts`.Also there could be some other methods which should be placed within our `form.service`.So try creating a separate service for firebase and then import it in our form-service

Reply:
I will rename it soon.